processing/src/main/java/io/druid/query/aggregation/post/ArithmeticPostAggregator.java

```diff
@@ -132,30 +154,37 @@ public String toString()
   {
     PLUS("+")
         {
-          double compute(double lhs, double rhs)
+          public double compute(double lhs, double rhs)
           {
             return lhs + rhs;
           }
         },
     MINUS("-")
         {
-          double compute(double lhs, double rhs)
+          public double compute(double lhs, double rhs)
           {
             return lhs - rhs;
           }
         },
     MULT("*")
         {
-          double compute(double lhs, double rhs)
+          public double compute(double lhs, double rhs)
           {
             return lhs * rhs;
           }
         },
     DIV("/")
         {
-          double compute(double lhs, double rhs)
+          public double compute(double lhs, double rhs)
           {
-            return (rhs == 0.0) ? 0 : (lhs / rhs);
+            return (rhs == 0.0) ? 0 : lhs / rhs;
+          }
+        },
+    QUOTIENT("quotient")
```

Comment:
hmmm, i think what we actually need is a having filter for metrics

in any case, can you add some docs and I can merge